tools/tests/test_modern_config.conf

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Instead of `modern` and falling into the Microsoft "new technology" pit, maybe we append a `v2`. So when we change/refactor again we can just keep adding new versions, subtly. :8ball: